app/controllers/concerns/two_factor_authenticatable_methods.rb

```diff
@@ -223,7 +223,7 @@ def unconfirmed_phone?
   end
 
   def phone_view_data
-    { confirmation_for_phone_change: confirmation_for_phone_change?,
+    { confirmation_for_add_phone: confirmation_for_add_phone?,
```

Comment:
where did this name change come from? Just for readability? Or does it have a particular use?

Reply:
It was changed to communicate that the method would be called when users don’t already have a phone associated with their account and are adding a new one. Previous method name was a bit misleading because it was being called in that context even though there was no phone. does that make sense?